File: identhub/src/main/java/de/solarisbank/identhub/contract/sign/ApplicationProcessingFragment.java
package de.solarisbank.identhub.contract.sign;

import de.solarisbank.identhub.R;
import de.solarisbank.identhub.di.FragmentComponent;
import de.solarisbank.identhub.progress.ProgressIndicatorFragment;

public class ApplicationProcessingFragment extends ProgressIndicatorFragment {

    @Override
    protected int getTitleResource() {
        return R.string.progress_indicator_precessing_application_title;
    }

    @Override
    protected int getMessageResource() {
        return R.string.progress_indicator_precessing_application_message;
    }

    @Override
    protected void inject(FragmentComponent component) {
        component.inject(this);
    }

    @Override
    protected void initViews() {
        super.initViews();
        progressBar.setOnClickListener(v -> {
            //todo could be navigated with deeplink
//            Intent intent = new Intent(getContext(), IdentitySummaryActivity.class);
//            startActivity(intent);
//            getActivity().finish();
        });
    }
}

File: identhub/src/main/java/de/solarisbank/identhub/contract/sign/ApplicationProcessingFragment.kt
package de.solarisbank.identhub.contract.sign

import android.view.View
import de.solarisbank.identhub.R
import de.solarisbank.identhub.di.FragmentComponent
import de.solarisbank.identhub.progress.ProgressIndicatorFragment

class ApplicationProcessingFragment : ProgressIndicatorFragment() {

    override fun getTitleResource(): Int {
        return R.string.identhub_progress_indicator_precessing_application_title
    }

    override fun getMessageResource(): Int {
        return R.string.identhub_progress_indicator_precessing_application_message
    }

    override fun inject(component: FragmentComponent) {
        component.inject(this)
    }

    override fun initViews() {
        super.initViews()
        progressBar!!.setOnClickListener { v: View? -> }
    }

}